src/client/services/board/persistence/persistence.js

```diff
@@ -70,17 +68,16 @@ var PersistenceService = SparkBase.extend({
   /**
    * Set a snapshot image for a board
    *
-   * @param {Conversation} conversation - the current conversation that the board belongs
    * @param {Board~Channel} channel
    * @param {File} image
    * @returns {Promise<Board~Channel>}
    */
-  setSnapshotImage: function setSnapshotImage(conversation, channel, image) {
+  setSnapshotImage: function setSnapshotImage(channel, image) {
     var imageScr;
-    return this.spark.board._uploadImage(conversation, image)
+    return this.spark.board._uploadImage(channel, image, {hiddenSpace: true})
       .then(function encryptScr(scr) {
         imageScr = scr;
-        return this.spark.encryption.encryptScr(imageScr, conversation.defaultActivityEncryptionKeyUrl);
+        return this.spark.encryption.encryptScr(imageScr, channel.defaultEncryptionKeyUrl);
```

Comment:
Can we just do:
.then((scr) => this.spark.encryption.encryptScr(scr, channel.defaultEncryptionKeyUrl))